handlers/twitter.js

```diff
@@ -1,23 +1,39 @@
-const Twitter = require('twitter')
+const Twitter = require('twit')
 const morph = require('morphmorph')
 
 const RATE_LIMIT_CODE = 420
+const MAX_ALT_TEXT_LENGTH = 420
```

Comment:
Funny these two values are the same . . .

Reply:
I cannot imagine it was happenstance